src/index.js

```diff
@@ -104,6 +105,30 @@ const errorSignature = err =>
 
 const getErrorCode = _.pipe(_.get('ruleId'), _.split('/'), _.last);
 
+const preprocessorPlaceholder = '___';
+const preprocessorTemplate = `JSON.stringify(${preprocessorPlaceholder})`;
```

Comment:
Is JSON.stringify even necessary? Just wrapping the JSON is parentheses would make for valid JS: `({"foo":1})`

Reply:
Yes, but as soon as additional eslint rules kick in (for example `no-unused-expressions`) or prettier tries to parse the file with babel, things start to fall apart. One could work around those issues by fine-tuning the eslintrc file, but just adding `JSON.stringify` simplifies things in practice a lot.